proto/desmos/supply/v1/query.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto3";
+package desmos.supply.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v3/x/supply/types";
+
+
+// Query defines the gRPC querier service.
+service Query {
+
+  // TotalSupply queries the total supply of the given denom, converted with the given divider
+  rpc TotalSupply(QueryTotalSupplyRequest)
+      returns (QueryTotalSupplyResponse) {
+    option (google.api.http).get = "/desmos/supply/v1/total-supply/{denom}/{divider}";
+  }
+
+  // CirculatingSupply queries the circulating supply (Total supply - community pool - vesting tokens) of the given
+  // denom, converted with the given divider
+  rpc CirculatingSupply(QueryCirculatingSupplyRequest)
+      returns (QueryCirculatingSupplyResponse) {
+    option (google.api.http).get = "/desmos/supply/v1/circulating-supply/{denom}/{divider}";
```

Comment:
As above, the `divider` can be a query parameter rather than a URL parameter:
```suggestion
    option (google.api.http).get = "/desmos/supply/v1/circulating-supply/{denom}";
```